rotkehlchen/api/v1/schemas.py

```diff
@@ -4065,3 +4065,32 @@ def validate_schema(
                         message=f'Account {address} with chain {evm_chain.to_name()} is not tracked by rotki',  # noqa: E501
                         field_name='address',
                     )
+
+
+class AddressesInteraction(Schema):
+    from_address = fields.String(required=True)
```

Comment:
we have `EvmAddressField()` so perhaps it makes sense to use that here and below.